data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/configuration/ExportConfig.java

```diff
@@ -33,4 +37,15 @@ public Region getAwsRegion() {
         return s3Region != null ? Region.of(s3Region) : null;
     }
 
+    public String getS3SseKmsKeyId() {
+        return s3SseKmsKeyId;
+    }
+
+    @AssertTrue(message = "KMS Key ID must be a valid one.")
+    boolean isKmsKeyIdValid() {
+        // If key id is provided, it should be in a format like
+        // arn:aws:kms:us-west-2:123456789012:key/0a4bc22f-bb96-4ad3-80ca-63b12b3ec147
+        return s3SseKmsKeyId == null || Arn.fromString(s3SseKmsKeyId).resourceAsString() != null;
```

Comment:
We should probably wrap this in a try catch to avoid NPE's and exceptions from just throwing here with invalid ARN format, but this can be added as a follow on